pkg/infinispan/cluster.go

```diff
@@ -50,6 +55,8 @@ type ClusterInterface interface {
 	CacheNames(podName string) ([]string, error)
 	GetMetrics(podName, postfix string) (*bytes.Buffer, error)
 	GetCacheManagerInfo(cacheManagerName, podName string) (map[string]interface{}, error)
+	GetLoggers(podName string) (map[string]string, error)
+	SetLogger(podName, loggerName, loggerLevel string) error
```

Comment:
Should we use `Logger` here for the parameters and return type?

Reply:
Yes and no. It could be get additional a value, if pass/get objects are the same structure. But in our case we need to transform Logger to the plain text path in the request and back to the map[string]string in response